docs/data/base/guides/next-js-app-router/next-js-app-router.md

```diff
@@ -0,0 +1,97 @@
+# Next.js App Router
+
+<p class="description">Learn how to use Base UI with the Next.js App Router.</p>
+
+:::info
+Starting fresh on a new App Router-based project?
+
+Jump right into the code with this [example repo](https://github.com/mui/material-ui/blob/master/examples/base-next-app-router-ts).
+:::
+
+## Next.js and React Server Components
+
+The Next.js App Router implements React Server Components, a [new feature](https://github.com/reactjs/rfcs/blob/main/text/0227-server-module-conventions.md#changes-since-v1) introduced in React 18.
+
+To support the App Router, currently all components and hooks from Base UI and other MUI libraries are exported with the `"use client"` directive.
+
+:::warning
+React Server Components does not replace, and is separate from server-side rendering (SSR). Client Components are still server-rendered to HTML.
+
+For more details, see this [explanation](https://github.com/reactwg/server-components/discussions/4) by the React team.
+:::
+
+## Setting up Base UI with the App Router
+
+Base UI is styling-agnostic; setting up a Next.js App Router project largely depends on the styling solution:
+
+### Emotion
+
+If you're using Emotion, or something Emotion-based like MUI System, you can follow the [same steps](/material-ui/guides/next-js-app-router/#using-material-ui-with-a-custom-theme) outlined for Material UI.
+
+### Tailwind CSS
+
+[Install and initialize](https://tailwindcss.com/docs/guides/nextjs) Tailwind CSS, be sure to add the `app` directory and other directories to `tailwind.config.js`
```

Comment:
To make the link more informative—readers should know exactly what they'll find if they click the link.

```suggestion
Follow the [Tailwind CSS doc on working with Next.js](https://tailwindcss.com/docs/guides/nextjs), and be sure to add the `app` directory and other directories to `tailwind.config.js`, as shown below:
```